halpybot/commands/caseutils.py

```diff
@@ -746,5 +742,35 @@ async def cmd_editnote(ctx: Context, args: List[str], case: Case):
     case.case_notes[
         note_index
     ] = f"{' '.join(args[2:])} - {ctx.sender} ({now(tz='UTC')})"
-    await ctx.bot.board.mod_case_notes(case_id=case.board_id, new_notes=case.case_notes)
     return await ctx.reply(f"Notes for case {case.board_id} updated.")
+
+
+# ADMINSTRATIVE MANAGEMENT
+@Commands.command("delcase", "remcase")
+@needs_permission(Admin)
+@in_channel
+@gather_case(1)
+async def cmd_remcase(ctx: Context, args: List[str], case: Case):
+    """
+    Removes a case from the board administratively, in case something screwed up.
+    Not to be done lightly!
+
+    Usage: !remcase [board ID]
+    Aliases: delcase
+    """
+    if not args:
+        new_notes = (
+            f"Admin Case Termination: {' '.join(args[1:])} - {ctx.sender}"
+            f" ({now(tz='UTC').to_time_string()})"
+        )
+    else:
+        new_notes = (
+            f"{ctx.sender} administratively removed case {case.board_id} "
+            f"from the board at {now(tz='UTC').to_time_string()}"
+        )
```

Comment:
possible logic error: it looks like the if condition is inverted, based on how `args` is used in the top branch.